packages/nimble-components/src/utilities/directive/dynamic-ref.ts

```diff
@@ -0,0 +1,62 @@
+import {
```

Comment:
Does FAST have unit tests for their RefBehavior that we could leverage for patterns to write unit tests for this class?

Reply:
Not that I've found, but writing our own should be pretty quick anyway.